src/types.ts

```diff
@@ -0,0 +1,16 @@
+export interface Project {
+  name: string;
+  authors: { [author: string]: string }[];
+  description: string;
+  paratimes: string[];
+  languages: string[];
+  screenshots: string[];
+  license: string;
+  tags: string[];
+  codeUrl: string;
+  demoUrl: string;
+  tutorials?: { [tutorial: string]: string }[]; 
```

Comment:
This type is odd. @matevz https://github.com/oasisprotocol/playground/blob/df30664/projects/demo-voting/description.yaml#L20-L21 this means `tutorials: [ { "Official Oasis Documentation": "https://docs.oasis.io/dapp/sapphire/gasless" } ]`.

Did you mean `tutorials: { "Official Oasis Documentation": "https://docs.oasis.io/dapp/sapphire/gasless" }`

Reply:
This should be an ordered list of Title -> URL elements.

Reply:
> Did you mean `tutorials: { "Official Oasis Documentation": "https://docs.oasis.io/dapp/sapphire/gasless" }`

This probably doesn't preserve ordering, right?

Reply:
In most cases order is preserved. One exception I can think of are numbers `2: https://`